### DISCUSSION TITLE
Phone set in Basque CV dictionary v2.0.0

### DISCUSSION THREAD

**Discussion by jpereznavarro** [MontrealCorpusTools/mfa-models#23] [Pronunciation dictionaries]
Hi there!

I'm working on a Basque-Spanish bilingual processing project, using the MFA as aligner. First of all, thank you very much for this wonderful tool, it's really impressive how useful it is for our research.

I have a question regarding how the phone sets were set for the [Basque CV dictionary v2.0.0](https://mfa-models.readthedocs.io/en/latest/dictionary/Basque/Basque%20CV%20dictionary%20v2_0_0.html#basque-cv-dictionary-v2-0-0) and the [Spanish (Spain) MFA dictionary v2.0.0](https://mfa-models.readthedocs.io/en/latest/dictionary/Spanish/Spanish%20%28Spain%29%20MFA%20dictionary%20v2_0_0.html#spanish-spain-mfa-dictionary-v2-0-0). In theory the phone sets should be fairly similar (but for a few sounds like [ts̺] or [ts̻] that are only present in Basque). However, the phone set of the Spanish pronunciation dictionary is considerably larger than Basque. For instance, it contains the [w] phone as an adaptation of the /u/ phoneme, while the Basque dictionary does not include it and this adaptation also takes place in Basque.

Given that we are investigating bilingual sound processing, we would like to have as comparable as possible phone sets across both languages. Is there anyway of retraining or adapting the acoustic models so they allow for fairly similar phone sets? Any piece of information on how the phone sets are achieved would also be appreciated.

Thank you very much!

**Comment by emilyahn** [comment]
Hi Jose,
In terms of the Basque CV dictionary, we used the [XPF](https://cohenpr-xpf.github.io/XPF/Convert-to-IPA.html) tool to convert Basque text into phone sequences. Here is [documentation](https://cohenpr-xpf.github.io/XPF/conv_resources/info/eu.html) of their inventory.

Here are a couple ways I can think of answering your last questions.

1. (Easier) This [tool](https://github.com/jhdeov/interlingual-MFA) helps apply acoustic models from one language to another language that has a different phone set. 
2. (Harder) If there are certain phonological rules you would like to apply to the Basque dictionary (e.g. adding an alternate [w] phone), you may try to retrain the Basque Common Voice corpora yourself by the following
  * modify the [lexicon](https://osf.io/t957v/files/osfstorage/64b28a88015bbe0975eee8cd) from our corpus
  * download Basque audio and text from [Common Voice](https://commonvoice.mozilla.org/en/datasets) (take care that it's Version 7.0!!)
  * follow these [instructions](https://github.com/echodroff/vox-communis-2022-lrec) on training your own acoustic models

**Comment by mmcauliffe** [comment]
Emily, correct me if I'm wrong, but there's also no Spanish dictionaries/models as part of Vox Communis, right?  

The MFA phone set seeks to be a bit more narrow and phonetic (and it's based on community generated transcriptions via [Wikipron](https://github.com/CUNY-CL/wikipron/tree/master/data)), so that's why it's going to be larger than a strictly phonemic phone set than more rule based XPF and/or Epitran produces.  You can see the general MFA phone set details and Spanish specific notes here: https://mfa-models.readthedocs.io/en/latest/mfa_phone_set.html#spanish.

If you want to train a new Spanish model with a broader transcription (i.e., using the [Spanish XPF](https://cohenpr-xpf.github.io/XPF/conv_resources/info/es.html) rules to generate a dictionary), you can see the data sources used in training here: https://mfa-models.readthedocs.io/en/latest/acoustic/Spanish/Spanish%20MFA%20acoustic%20model%20v2_0_0a.html.

**Reply by emilyahn** [reply]
Right, we didn't touch Spanish for our Vox Communis dataset since it was so large.

**Comment by jpereznavarro** [comment]
Thank you Emily and Michael for your replies, I have a clearer idea now of how I should proceed. I might come back here if I get stuck at some point. Thanks!
